js/webform_civicrm_admin.js

```diff
@@ -451,11 +451,11 @@ var wfCiviAdmin = (function ($, D) {
             if (cl[1] == 'icon') {
               var icon_name = cl[2];
               if (cl[0] == 'contact') {
-                name = 'name="' + (i + 1) + '_contact_type"'
+                name = 'name="' + (i + 1) + '_contact_type"';
                 var type = $('select[name="' + (i + 1) + '_contact_type"', '#webform-civicrm-settings-form').val();
                 icon_name = getContactIcon(type);
               }
-              $('#webform-civicrm-settings-form .vertical-tabs__menu-item a').eq(i).children('span').prepend('<i class="crm-i ' + icon_name + '" ' + name + '> </i>');
+              $('#webform-civicrm-settings-form .vertical-tabs__menu-item').eq(i).children('a').prepend('<i class="crm-i ' + icon_name + '" ' + name + '> </i>');
```

Comment:
@KarinG yes I see on this line you are using `crm-i` fontawesome icons. But when I tested on a fresh copy of D9 these icons did not actually show up without this patch.

Reply:
So very odd... are you on D9.3? I'm looking at a D9.2.9 site and can see the icons. 

I'm going to test this manually as we don't have much test coverage yet for the admin form.

Reply:
Looks like `civicrm-buildkit` gave me Version 9.1.15.
I'm not sure why that would make a difference though.
At any rate this change ought to make the jquery less brittle so if the icons still work for you after as well as before, it's probably good to merge.

Reply:
I'm still going to test it - I'm testing everything!